spock-core/src/main/java/org/spockframework/runtime/extension/builtin/PendingFeatureInterceptor.java

```diff
@@ -12,7 +12,7 @@
   public void intercept(IMethodInvocation invocation) throws Throwable {
     try {
       invocation.proceed();
-    } catch (AssertionError e) {
+    } catch (Throwable t) {
```

Comment:
I agree, but `invocation.proceed()` is caught like this in `AutoCleanupInterceptor`, `BaseSpecRunner`, `FailsWithInterceptor`, `FailsWithInterceptor`, `TimeoutInterceptor` and `UnitilsInterceptor` also